scss/_print.scss

```diff
@@ -18,18 +24,37 @@
     text-decoration: underline;
   }
 
+  a[href]::after {
+    content: " (" attr(href) ")";
+  }
+
   abbr[title]::after {
     content: " (" attr(title) ")";
   }
 
+  //
+  // Don't show links that are fragment identifiers,
+  // or use the `javascript:` pseudo protocol
+  //
+
+  a[href^="#"]::after,
+  a[href^="javascript:"]::after {
+    content: "";
+  }
+
   pre,
   blockquote {
-    border: $border-width solid #999;
+    border: 1px solid #999;
```

Comment:
This was removed so that we are on par with upstream and since it wasn't marked as custom. I can put it back and add a comment indicating it's custom to fit our needs.